docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -28,23 +28,32 @@ following order:
 1. `init`: A new certificate authority is issued, but not used.
 1. `update_clients`: The Teleport Auth Service uses the new CA to sign
    certificates but continues to trust certificates signed by the original CA.
-1. `update_servers`: Teleport cluster components (Agents, Auth Service, and
-   Proxy Service instances) reload and start serving TLS and SSH certificates
-   signed by the new certificate authority, but still accept certificates issued
-   by the original certificate authority. This only applies to the Teleport
-   [`host` CA](#host).
-   1. `standby`: No rotation in progress. All operations have completed.
+1. `update_servers`: Any server components in the cluster that accept incoming
+   connections from clients should reload their identity and start serving
+   certificates issed by the new CA, but still accept certificates issued by the
+   original CA.
+   When rotating the [`host` CA], this automatically applies for teleport
+   cluster components (Agents, Auth Service, and Proxy Service instances).
+1. `standby`: No rotation in progress. All operations have completed.
 
 Before the final `standby` phase, you can also put the rotation in the
-`rollback` phase, aborting the rotation and returning to the original
-certificate authority.
+`rollback` phase to abort the rotation return to the original certificate
+authority.
+After the `rollback` phase you will then proceed to the `standby` phase.
 
 CA rotations can be **manual** or **semi-automatic**. In manual mode, admins
 must instruct the Teleport Auth Service to advance from one phase to the next.
 Between phases, admins can prepare their infrastructure to adjust to each
 change. In semi-automatic mode, the Teleport Auth Service cycles through each
 phase automatically, with a grace period between each phase.
 
+In 17.1.0+ `tctl auth rotate` (with no arguments) starts an interactive
```

Comment:
In this change, we mention `tctl auth rotate` after the manual and semi-automatic options. Would it make sense to structure the guide around the wizard instead, and leave the documentation of the arguments for our reference guides (i.e., this guide would be the fast path)? Otherwise, while the argumentless form of `tctl auth rotate` is a simplification, the documentation actually becomes a little more complex.